packages/console-types/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## Unreleased
```

Comment:
we use changeset in this repo, please discard this changes and run `pnpm changeset`